src/NzbDrone.Core/Http/CloudFlare/CloudFlareCaptchaException.cs

```diff
@@ -0,0 +1,19 @@
+using NzbDrone.Common.Exceptions;
+using NzbDrone.Common.Http;
+
+namespace NzbDrone.Core.Http.CloudFlare
+{
+    public class CloudFlareCaptchaException : NzbDroneException
+    {
+        public HttpResponse Response { get; set; }
+
+        public CloudFlareCaptchaRequest CaptchaRequest { get; set; }
+
+        public CloudFlareCaptchaException(HttpResponse response, CloudFlareCaptchaRequest captchaRequest)
+            : base("Unable to access {0}, blocked by CloudFlare CAPTCHA. Likely due to shared-IP VPN.", response.Request.Url.Host)
```

Comment:
`Likely due to shared-IP VPN.` - could also be a VPS, do we need to specify likely part at all?